src/collection-browser.ts

```diff
@@ -165,6 +165,11 @@ export class CollectionBrowser
    */
   @property({ type: Boolean }) isManageView = false;
 
+  /**
+   * If facet-top view UI active
+   */
+  @property({ type: Boolean }) isFacetTopView = false;
```

Comment:
we need to rename these to say isProfiePage or something like this

Reply:
q: why do we need this boolean at all? the named facet should exist as slot element.

Reply:
same as the comment below... were going to use something generic for now to decouple feature name from the actual behavior for now